docs/content/filtering.md

```diff
@@ -764,8 +764,6 @@ E.g. for the following directory structure:
 The command `rclone ls --exclude-if-present .ignore dir1` does
 not list `dir3`, `file3` or `.ignore`.
 
-`--exclude-if-present` can only be used once in an rclone command.
```

Comment:
Perhaps explicitly say that the flag can be repeated multiple times?

Reply:
Good idea. Pushed a new commit with suggestion. I found it a bit hard to come up with a concise sentence for it, so if you have any better off the top of your head let me know.

Reply:
I like your sentence, except for misspelled  presence :-)